app/components/modals/CantCloseModals/CantCloseModals.jsx

```diff
@@ -14,8 +15,10 @@ const CantCloseModals = (props) => {
     onHideCantCloseModal,
     shutdownApp,
     accountMixerRunning,
-    purchasingTickets
+    purchasingTickets,
+    dexOrdersOpen
   } = useCantCloseModal();
+
   let Component = () => <></>;
```

Comment:
this can be be wrapped with useMemo to memorize the value of `Component` and avoid extra unnecessary 
evaluations.